rules/ampedwireless.com.xml

```diff
@@ -0,0 +1,12 @@
+<sitename name="ampedwireless.com">
+  <docname name="Terms and Conditions">
+    <url name="https://www.ampedwireless.com/legal#terms" xpath="//div[@id='terms']" lang="EN">
+      <norecurse name="arbitrary"/>
+    </url>
+  </docname>
+  <docname name="Privacy Policy">
+    <url name="https://www.ampedwireless.com/legal#privacys" xpath="//div[@id='privacy']" lang="EN">
```

Comment:
Crawler isn't able to find this one. The [linked page](https://www.ampedwireless.com/legal#privacy) hides the specified element until you click on something in the side bar, but it should be in the DOM regardless. Any tips?

Reply:
Ah, `//*[@id='privacy']` works.

Reply:
great!